### DISCUSSION TITLE
posts/bikes/

### DISCUSSION THREAD

**Discussion by giscus** [Shitao5/shitao-blog#52] [Comments]
# posts/bikes/

8 月 26 日，入手了俺的第一辆公路车，横穿大半个杭州城，从车店骑回家。睡前不禁想到从小到大那些骑过的自行车。全盘扫描了自己的回忆，有三辆让我印象深刻。
[&hellip;] 对于土生土长的农村娃，自行车是和其他小伙伴开疆拓土的重要工具，但法律规定「驾驶自行车必须年满 12 周岁」，所以 12 岁前我并没有自己的自行车，好在姐姐的自行车成了我童年时期的非专属座驾。我已记不清它的具体样子，只好根据 &hellip;

https://shitao5.org/posts/bikes/

**Comment by achuanya** [comment]
帅

**Reply by Shitao5** [reply]
嘿嘿，骑友好！
